<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWidget</class>
 <widget class="QDockWidget" name="ControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>658</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>210</width>
    <height>240</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(181, 181, 181);</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QRadioButton" name="radioButtonZones">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Zones</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupView</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButtonAccessibility">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Accessibility</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupView</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButtonLandValue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Land value</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupView</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButtonNeighborPopulation">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Neighbor population</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupView</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButtonPollution">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Pollution</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupView</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButtonSlope">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Slope</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupView</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButtonPopulation">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Population</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupView</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButtonCommercialJobs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Commercial jobs</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupView</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButtonIndustrialJobs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Industrial jobs</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupView</string>
     </attribute>
    </widget>
    <widget class="QSlider" name="horizontalSliderOpacity">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>300</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Opacity:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonAll">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>All</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupView</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButtonLife">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Life</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupView</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButtonShop">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Shop</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupView</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButtonFactory">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Factory</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupView</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButtonNeighborCommercial">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Neighbor commercial</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupView</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>seed:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRandomSeed">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>351</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>346</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Initialize</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Run Simulation</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>steps:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditNumSteps">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSaveScores">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>save scores</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>move rate:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMoveRate">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSaveZonings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>save all zonings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonNextStep">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Next Step</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSaveBestZoning">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>save best zoning</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonRandomGeneration">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>570</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Random Generation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNumRandomGeneration">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>540</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>541</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string># numbers:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupView"/>
 </buttongroups>
</ui>
